src/common/arrstr.cpp

```diff
@@ -573,3 +575,170 @@ wxArrayString wxSplit(const wxString& str, const wxChar sep, const wxChar escape
 
     return ret;
 }
+
+
+#ifdef wxUSE_REGEX
+
+wxRegEx naturalNumeric(_T("[0-9.]+"));
```

Comment:
Please don't use `_T()` in the new code. `wxT()` or, more likely, `wxS()` should be used instead but most often you should just omit it entirely. Or use `L"..."`.